examples/gno.land/r/gnoland/faucet/faucet.gno

```diff
@@ -64,22 +63,20 @@ func Render(path string) string {
 	if gInPause {
 		output += "Status: inactive.\n"
 	} else {
-		output += "Status: active.\n\n"
+		output += "Status: active.\n"
 	}
 	output += ufmt.Sprintf("Balance: %s.\n", balance.String())
-	output += ufmt.Sprintf("Total transfer: %s (in %d times).\n\n", gTotalTransferred.String(), gTotalTransfers)
+	output += ufmt.Sprintf("Total transfers: %s (in %d times).\n\n", gTotalTransferred.String(), gTotalTransfers)
```

Comment:
Is there a possibility to use `bytes.Buffer` or `strings.Builder` for this string concatenation?

Appending strings like this is really inefficient, when you have access to standard library helpers.

I've benchmarked these three options, you can try running them locally as well to see results for your machine:
https://goplay.tools/snippet/yHgXTpmG1FQ

Reply:
I definitely agree with it if this is written in go. But gno is written as an interpreted language.  We get more parsed Types, Values, and Blocks from importing strings.gno code in avl store before it is executed.  In this use case, we do not persistent rendered results. it is gas-saving and efficient to